Dockerfile.mysql

```diff
@@ -0,0 +1,12 @@
+FROM mysql:latest
+
+ENV MYSQL_ROOT_PASSWORD=1234
```

Comment:
Hardcoding the MySQL root password is a security risk. Consider using environment variables or Docker secrets to manage sensitive information securely.